ansible/roles/common/defaults/main.yaml

```diff
@@ -6,3 +6,13 @@ bin_dir: /usr/bin
 # The directory used by Ansible to temporarily store
 # files on Ansible managed systems.
 ansible_temp_dir: /tmp/.ansible/files
+
+# The director used for the docker config
+docker_config_dir: "/etc/sysconfig"
+
+#Some default values that may get modified in etc/tasks/main.yml
+is_atomic: false
+is_coreos: false
+has_rpm: false
+has_firewalld: false
+has_iptables: false
```

Comment:
Nice. Is the list complete or are there any other facts worth setting to default value?